docs/source/usage.rst

```diff
@@ -8,3 +8,28 @@ Installation
 
 
 Under Construction!
+
+Application settings
+--------------------
+
+Settings for the application are shipped with the package in the `application.yml <https://github.com/neutrons/SNAPRed/blob/next/src/snapred/resources/application.yml>`_ file.
+There is also a `test application.yml <https://github.com/neutrons/SNAPRed/blob/next/tests/resources/application.yml>`_ file used in tests.
+Most of what is included in the settings is a parameterization of options for the instrument.
+It may be desireable to override some of the settings which can be done via environment injection.
+For example
+
+.. code-block:: yaml
+   :caption: Example environment ``myenv.yml``
+
+   environment: myenv
+
+   instrument:
+     home: ~/snapred-testing/
+
+Can be used to override the ``instrument.home`` directory to ``~/snapred-testing/``.
+At this time, only the ``instrument.home`` and ``samples.home`` directories will expand ``~/`` to a directory.
+This file can be supplied via command line injection
+
+.. code-block:: sh
+
+   env=myenv.yml snapred
```

Comment:
Sorry for not catching this sooner, but why is the `env` including the file extension?  The environment file is derived from the environment and will always be a `.yml`.  Not that we have other things dependent on `env` right now, but potentially.

Reply:
It can also be called as `env=myenv snapred` to the same effect. The previous code always added on the `.yml` extension and looked into the resource directory. This way allows for a varient of specifying a relative or absolute file path.

Reply:
Ah ok sounds good.